File: features/lead/src/main/java/com/nivasafinance/features/leadlender/dto/RejectLeadLenderRequest.java
package com.nivasafinance.features.leadlender.dto;

import jakarta.validation.constraints.NotNull;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class RejectLeadLenderRequest {
    
    @NotNull(message = "Reject reason is required")
    private String rejectReason;

    private String remarks;
}

File: features/lead/src/main/java/com/nivasafinance/features/leadlender/dto/RejectLeadLenderRequest.kt
package com.nivasafinance.features.leadlender.dto

import com.nivasafinance.features.leadlender.enum.RejectReason
import jakarta.validation.constraints.NotNull

data class RejectLeadLenderRequest(
    @field:NotNull(message = "Reject reason is required")
    val rejectReason: RejectReason
)
